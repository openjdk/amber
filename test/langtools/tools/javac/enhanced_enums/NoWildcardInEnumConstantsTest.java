/*
 * @test /nodynamiccopyright/
 * @bug 8177505
 * @summary wildcard is not allowed in enhanced enums
 * @compile/fail/ref=NoWildcardInEnumConstantsTest.out -XDrawDiagnostics NoWildcardInEnumConstantsTest.java
 */

enum NoWildcardInEnumConstantsTest<X> {
    A<?>
}
