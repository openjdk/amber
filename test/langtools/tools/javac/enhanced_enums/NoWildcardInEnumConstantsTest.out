NoWildcardInEnumConstantsTest.java:9:5: compiler.err.type.found.req: ?, (compiler.misc.type.req.exact)
1 error
